harmony/harmony.py

```diff
@@ -361,6 +362,57 @@ def _params(self, request: Request) -> dict:
 
         return params
 
+    def _headers(self) -> dict:
+        """
+        Create (if needed) and return a dictionary of headers.
+        Code partially adaped from:
+            https://github.com/requests/toolbelt/blob/master/requests_toolbelt/utils/user_agent.py
+        """
+        if 'headers' not in self.__dict__:
+            session = self._session()
+            existing_user_agent_header = session.headers.get('User-Agent')
+            if existing_user_agent_header:
+                user_agent_content = set([existing_user_agent_header])
+            else:
+                user_agent_content = set([])
+
+            # Get harmony package info
+            try:
+                import harmony
```

Comment:
I think in most of our Python code we've been keeping our imports at the top of the file rather than inline.

Reply:
Thanks for you comment Chris. It's probably a little bit strange to import harmony inside harmony, and I imported harmony here simply to get the version info, so I just put it inside this try block.  I can move it to the top if it doesn't seem to be strange or won't break things, or ideally, if there is a better way to get the harmony version info that will be great.

Reply:
I don't really know Python and have no strong opinions. Maybe someone more knowledgable can weigh in.